packages/lpa-questionnaire-web-app/src/views/development-plan.njk

```diff
@@ -0,0 +1,118 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "components/appeal-details/macro.njk" import appealDetails %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: "Back",
+    href: backLink,
+    attributes: {
+      'data-cy': 'back'
+    }
+  }) }}
+{% endblock %}
+
+{% block pageTitle %}
+  Development Plan Document or Neighbourhood Plan - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK
+{% endblock %}
+
+{% block content %}
+
+  {% if errors %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errorSummary
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form action="" method="POST" novalidate>
+        <span class="govuk-caption-l">
+          <span class="govuk-visually-hidden">Section. </span>
+          Optional supporting documents
+        </span>
+
+        {% call govukFieldset({
+          legend: {
+            text: "Development Plan Document or Neighbourhood Plan",
+            classes: "govuk-fieldset__legend--l",
+            isPageHeading: true
+          }
+        }) %}
+        <p>Has any Development Plan Document or Neighbourhood Plan relevant to this appeal been submitted for examination (or in the case of a Neighbourhood Plan, been examined and is awaiting a referendum)?</p>
+
+        {% set developmentPlanTextArea %}
+          {{ govukTextarea({
+            hint: {
+              text: "Tell us about any substantive changes in the progress of the emerging plan, and their relevance to this appeal (if the plan may not be adopted before the Inspector issues their decision)."
+            },
+            idPrefix: "plan-changes-text",
+            name: "plan-changes-text",
+            attributes: {
+              'data-cy': 'plan-changes-text'
+            },
+            value: values['plan-changes-text'],
+            errorMessage: errors['plan-changes-text'] and {
+                text: errors['plan-changes-text'].msg,
+                attributes: {
+                  'data-cy': 'plan-changes-error'
+                }
+            }
+          }) }}
+        {% endset %}
+
+        {{ govukRadios({
```

Comment:
This is missing a label - think we will have GDS issues without one

Reply:
In the preview there's only labels for the yes and the no in the component. Do we have to look into changing the component appearance from the spec?

Reply:
Changed with previous fix